portfolio/src/main/webapp/script.js

```diff
@@ -63,28 +48,32 @@ showSlides(slideIndex);
 
 // Next/previous controls
 function plusSlides(n) {
-  showSlides(slideIndex += n);
+  showSlides((slideIndex += n));
```

Comment:
Wow, I don't think I've seen this style of "increment and give me the result" before.